test/compose.c

```diff
@@ -1130,12 +1131,17 @@ main(int argc, char *argv[])
     test_modifier_syntax(ctx);
     test_include(ctx);
     test_override(ctx);
-    test_traverse(ctx, quickcheck_loops);
     test_string_length(ctx);
     test_decode_escape_sequences(ctx);
     test_encode_escape_sequences(ctx);
     test_roundtrip(ctx);
 
+    /* Turn off noisy deprecated keysym warnings from here. */
+    xkb_context_set_log_level(ctx, XKB_LOG_LEVEL_INFO);
+    xkb_context_set_log_verbosity(ctx, 0);
```

Comment:
We may also consider to actually fix them at some point 😄

Reply:
See #625

Reply:
I removed this part since it's no longer needed.